frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -295,6 +296,9 @@ private void handleRegisterModel(ChannelHandlerContext ctx, QueryStringDecoder d
         String handler = NettyUtils.getParameter(decoder, "handler", null);
         int batchSize = NettyUtils.getIntParameter(decoder, "batch_size", 1);
         int maxBatchDelay = NettyUtils.getIntParameter(decoder, "max_batch_delay", 100);
+        int initialWorkers = NettyUtils.getIntParameter(decoder, "initial_workers", 0);
```

Comment:
Why not just make one more API? This seems to overload the existing API with multiple way's to do some work.